.github/workflows/github_actions_aws_win_py32.yml

```
@@ -0,0 +1,20 @@
+name: nidaqmxbot_win_32
+
+on:
+  pull_request:
+    branches:
+      - master
```

Call this from `PR.yml`. 

Change the trigger:
```suggestion
on:
  workflow_call:
  workflow_dispatch:
```

and add a job to `PR.yml`:
```
  run_system_tests:
    uses: ./.github/workflows/run_system_tests.yml
    needs: [build, run-unit-tests]
```